frontend/public/actions/ui.ts

```diff
@@ -282,6 +284,20 @@ export const sortList = (
 
   return action(ActionType.SortList, { listId, field, func, orderBy });
 };
+export const setTableColumns = (id: string, selectedColumns: Set<string>) => {
+  let currentColumns = {};
+  try {
+    currentColumns = JSON.parse(localStorage.getItem(COLUMN_MANAGEMENT_LOCAL_STORAGE_KEY)) || {};
+  } catch (e) {
+    // Error parsing the data, do not store the current filters
+    /* eslint-disable-next-line no-console */
+    console.error('Error parsing column filters from local storage', e);
+    currentColumns = {};
```

Comment:
I don't think this is necessary since it was initialized to `{}` above.

Reply:
removed this